### PR TITLE
Fix imported package names to not use mixedCaps or under_scores

### PR DESCRIPTION
**Pull request by mqliang**
I mess things up when solve conflict after rebase this PR https://github.com/google/cadvisor/pull/934, and can not reopen the existing one, so I close it and recommit here.

### CODE REVIEW

File: api/handler.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/google/cadvisor/events"
-	httpMux "github.com/google/cadvisor/http/mux"
```
Comment (vishh) [inline api/handler.go]: Why not use mixedCaps? Is this change related to any golang conventions?

Comment (jimmidyson) [inline api/handler.go]: Yeah golang conventions

Comment (vishh) [inline api/handler.go]: Any pointers to this convention?

Comment (jimmidyson) [inline api/handler.go]: https://blog.golang.org/package-names#TOC_2.

Comment (vishh) [inline api/handler.go]: Thanks!

**Comment by vishh** [comment]
ok to test.

**Comment by vishh** [comment]
LGTM
